Tests/ClientTests/Messaging/GleanPlumbMessageTests.swift

```diff
@@ -23,38 +23,6 @@ class GleanPlumbMessageTests: XCTestCase {
         super.tearDown()
     }
 
-    // MARK: - Test isUnderExperimentWith
```

Comment:
If we're removing all these tests, and leaving an empty test file with just helpers and boilerplate, we may as well remove the whole file. :)